.travis.yml

```diff
@@ -7,12 +7,13 @@ cache:
     - node_modules
 
 scala:
-  - 2.11.8
-  - 2.12.4
+  - 2.13.2
 
 jdk:
   - oraclejdk8
```

Comment:
would it be hard to change this to use openjdk?

Reply:
not for now.

Reply:
no, just need to test it. We will go to openjdk 11+ at some point.